packages/mwp-api-state/src/sync/index.js

```diff
@@ -58,56 +58,53 @@ export function getDeprecatedSuccessPayload(successes, errors) {
  * @param {Object} routes The application's React Router routes
  * @returns {Function} an Epic function that emits an API_REQUEST action
  */
-export const getNavEpic = (routes, baseUrl) => {
-	const resolveRoutes = getRouteResolver(routes, baseUrl);
-	return (action, store) => {
```

Comment:
slight signature change, but mostly whitespace change - see https://github.com/meetup/meetup-web-platform/pull/433/files?w=1 to view without whitespace changes